src/components/spreadsheet/table-wrapper.tsx

```diff
@@ -487,10 +469,7 @@ export const TableWrapper: FunctionComponent<TableWrapperProps> = ({
                         currentNode={currentNode}
                         rowData={rowData}
                         columnData={reorderedColsDefs}
-                        fetched={
-                            equipments?.nodesId.find((nodeId) => nodeId === currentNode.id) !== undefined ||
-                            !!errorMessage
-                        }
+                        fetched={equipments?.nodesId.find((nodeId) => nodeId === currentNode.id) !== undefined}
```

Comment:
```suggestion
                        fetched={!isFetching}
```
maybe it should simply be that ?

Reply:
Yes it sounds equivalent